backend/Persistence/DB/Data/Test/Foods.json

```diff
@@ -164,7 +232,30 @@
         "Name": "Drink"
       }
     ],
+    "StartDecreasingAt": "2021-11-04T11:47:04.9955968+02:00",
+    "DecreaseType": "PERCENT",
```

Comment:
foods.json should have foods with decrease type of value as well, not only percent like right now